lib/main.ts

```diff
@@ -5,6 +5,8 @@ require('source-map-support').install();
 import ts = require("typescript");
 import fs = require("fs");
 
+export enum DART_KEYWORD_KIND { RESERVED = 0, BUILTIN = 1, LIMITED = 2 };
```

Comment:
Should this enum be closer to its use site? I think you can define it within the class body, can't you?